docs/release-notes/2.17.0.md

```diff
@@ -0,0 +1,74 @@
+# Percona Monitoring and Management 2.17.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">May 10, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- Custom Certificates support for MySQL monitoring, including remote monitoring, will let you monitor MySQL instances with such security restrictions. This functionality might help monitor Google Cloud SQL for MySQL instances easier.
+
+- External monitoring cleanups on UI to make it easier to use. You can fill all parameters or provide the URL of your endpoint, and PMM will detect all needed parts.
+
+- pg_stat_monitor 0.9.0 support. This change will give you compatibility with the latest version. More features coming later.
+
+- Single-line install of PMM Server on supported Linux distros (this feature is in Technical Preview).
+
+- DBaaS Changes: (this feature is in Technical Preview)
+    - Make it easier to preview DBaaS functionality, so users can quickly turn it ON on the Advanced settings page. ([Read more](../how-to/configure.md#advanced-settings))
+    - Database components management will enable PMM administrators to restrict some DB versions from creating DB Clusters.
+    - For PXC clusters created using DBaaS, the HAProxy will be used by default.  Please note: the proper monitoring HAProxy with DBaaS will come later.
+
+- Changes to *Sign in to Percona Platform*. From this release, Registration of the Percona account will be more secure and require additional confirmation.
+
+
+
+## New Features
+
+* [PMM-7863](https://jira.percona.com/browse/PMM-7863): DBaaS: Ability to specify in K8s configuration the version of HAProxy to be used for DB creation 
+* [PMM-7848](https://jira.percona.com/browse/PMM-7848), [PMM-7847](https://jira.percona.com/browse/PMM-7847), [PMM-7421](https://jira.percona.com/browse/PMM-7421): Add support for using SSL certificates between pmm-admin and monitored MySQL databases
+* [PMM-7883](https://jira.percona.com/browse/PMM-7883): Single-line install of PMM Server on supported Linux distros - [Technical Preview]
```

Comment:
Nothing to fix here but we have the "quickstart" which has LOTS of steps...marketing owns that I think...need to update that to reference this as new pmm-server install steps.